app/controllers/github_webhooks_controller.rb

```diff
@@ -0,0 +1,58 @@
+class GithubWebhooksController < ActionController::Base
+  skip_before_action :verify_authenticity_token
+  include GithubWebhook::Processor
+
+  # receive only IssuesEvent from github
+  def github_issues(payload)
+    case payload[:action]
+    when 'deleted'
+      destroy_author(payload[:issue])
+    when 'closed'
+      inactivate_author(payload[:issue])
+    when 'reopened'
+      activate_author(payload[:issue])
+    else
+      handle_issue(payload[:issue])
+    end
+    render json: {}, status: :ok
```

Comment:
I don't think you need to call `render` here, could you remove it to see if still works please?
Btw, the gem you are using is going to [set the header status](https://github.com/ssaunier/github_webhook/blob/1.1/lib/github_webhook/processor.rb#L69)

Reply:
Yes, you are right, thanks 👊 , I removed it.